usb/usb_host_hid/test/test_hid_basic.c

```diff
@@ -0,0 +1,516 @@
+/*
```

Comment:
Could we extend the tests? 
1. Test with multiple active interfaces (eg. keybour and mouse)
2. sudden disconnection test

Reply:
@tore-espressif  Yep, indeed. 

1. Just in case, I have all the devices with several active interfaces. How would you prefer this test to look like? 
2. Also, all disconnection for HID device is sudden. That was the one crucial idea, when I wrote the driver: HID device can appears or disappears any time. Or do you mean that disconnection should happen in the middle of the usb control transfer?

Reply:
1. Test with a device that has both Mouse and Keyboard interfaces. Access them both concurently and make sure that there is no 'cross-talk' . This would probably required updates of the mock HID device, so we can leave it for later. It would be nice to have it now, if it is easy to implement, though
2. Yes, start a lot of transfers in one thread and call `force_conn_state(false, 1);` from another thread

Reply:
Sudden disconnection test marked with ignore for now.